examples/kinsol/serial/kinLaplace_picard_kry.c

```diff
@@ -144,11 +144,11 @@ int main()
 
   infofp = fopen("KINSOL.log", "w");
 
-  retval = KINSetInfoFile(kmem, infofp);
-  if (check_retval(&retval, "KINSetInfoFile", 1)) return(1);
+  // retval = KINSetInfoFile(kmem, infofp);
+  // if (check_retval(&retval, "KINSetInfoFile", 1)) return(1);
 
-  retval = KINSetPrintLevel(kmem, 3);
-  if (check_retval(&retval, "KINSetPrintLevel", 1)) return(1);
+  // retval = KINSetPrintLevel(kmem, 3);
+  // if (check_retval(&retval, "KINSetPrintLevel", 1)) return(1);
```

Comment:
Can we just delete these lines instead of commenting them out?
```suggestion
```

Reply:
Yep, I meant to do so